memlab.bib

```diff
@@ -110,6 +110,52 @@ @string{vn
 @string{vr = {Vision Research}}
 
 
+@Article{SizeEtal18,
+  author  = {A E Sizemore and C Giusti and A Kahn and J M Vettel and R F Betzel and D S Bassett},
+  title   = {Cliques and cavities in the human connectome},
+  journal = {Journal of Computational Neuroscience},
+  year    = {2018},
+  volume  = {44},
+  number  = {1},
+  pages   = {115--145},
+}
+
+@Article{ReimEtal17,
+  author  = {M W Reimann and M Nolte and M Scolamiero and K Turner and R Perin and G Chindemi and P D\l{}otko and R Levi and K Hess and H Markram},
```

Comment:
This special character doesn't look quite right-- you need something in the curly braces, and I haven't seen "l" as a special character

Reply:
It's supposed to be the Polish 'l with stroke' ( ł ). TeX has it as just \l by default.
Wikibooks recommended using \l{} but the BibTeX guide recommends {\l} instead. I'll change to the latter because I think that's syntactically the clearest